paasta_tools/cli/cmds/spark_run.py

```diff
@@ -511,6 +511,9 @@ def emit_resource_requirements(spark_config_dict, paasta_cluster, pool, webui_ur
     }
     dimensions = {'framework_name': spark_config_dict['spark.app.name'], 'webui_url': webui_url}
 
+    constraints = dict(constraint.split(':') for constraint in spark_config_dict['spark.mesos.constraints'].split(';'))
+    pool = constraints['pool']
```

Comment:
Please remove possible trailing \ in the pool name.

Reply:
done